### PR TITLE
Add browser tool and wording edits

### PR DESCRIPTION
**Pull request by malcalakovalski**

### CODE REVIEW

File: haunted-house-2024.qmd [hunk 1]
```diff
@@ -55,11 +63,14 @@ We think existing NLP tools and emerging AI tools can **<span style="color:#1696
...
 - What are our blind spots?
 
 ## Demo Time!
-[Not sure if this should go at end or after text analysis sections Manu?]
-
+<div>
+```{=html}
+<iframe data-external="1" width="100%" height="500" src="topic-browser.html" title="Webpage example"></iframe>
+```
+</div>
```
Comment (judah-axelrod) [inline haunted-house-2024.qmd:75]: Looks awesome! Any chance we'd want to share the actual quarto site you made for the EAP work alongside this? Or do you feel like this gets the main functionality across?

File: haunted-house-2024.qmd [hunk 2]
```diff
@@ -55,11 +63,14 @@ We think existing NLP tools and emerging AI tools can **<span style="color:#1696
...
-- What are the exact use cases where we would *always* or *never* want to suggest an AI-based solution?
+- What use cases call for—or exclude—AI-based solutions?
 - What are our blind spots?
 
 ## Demo Time!
-[Not sure if this should go at end or after text analysis sections Manu?]
-
+<div>
+```{=html}
+<iframe data-external="1" width="100%" height="500" src="topic-browser.html" title="Webpage example"></iframe>
```
Comment (judah-axelrod) [inline haunted-house-2024.qmd:72]: Would we want this to just be full screen minus a title?

File: haunted-house-2024.qmd (hunk 1 above)
Comment (malcalakovalski) [reply haunted-house-2024.qmd:75]: I feel like this gets the point across and is a bit more compact. When I embedded the actual quarto site, it was a bit zoomed out. I can easily switch if you have a strong preference though!

File: haunted-house-2024.qmd (hunk 2 above)
Comment (malcalakovalski) [reply haunted-house-2024.qmd:72]: I think that would be ideal!

**Review by judah-axelrod — APPROVED** [review]

**Pull request merged**